injected/entry-points/apple.js

```diff
@@ -2,13 +2,20 @@
  * @module Apple integration
  */
 import { load, init } from '../src/content-scope-features.js';
-import { processConfig, isGloballyDisabled } from './../src/utils';
+import { processConfig, isGloballyDisabled, alwaysEnabledFeatures } from './../src/utils';
 import { isTrackerOrigin } from '../src/trackers';
 import { WebkitMessagingConfig, TestTransportConfig } from '../../messaging/index.js';
 
 function initCode() {
     // @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
-    const processedConfig = processConfig($CONTENT_SCOPE$, $USER_UNPROTECTED_DOMAINS$, $USER_PREFERENCES$);
+    const config = $CONTENT_SCOPE$;
+    // @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
+    const userUnprotectedDomains = $USER_UNPROTECTED_DOMAINS$;
+    // @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
+    const userPreferences = $USER_PREFERENCES$;
+
+    const processedConfig = processConfig(config, userUnprotectedDomains, userPreferences, alwaysEnabledFeatures);
```

Comment:
On the `processConfig` call site it's called `platformSpecificFeatures`, I don't really know why we pass it in anyway. I think we could just use it directly within utils.js to simplify the signature. (happy to be a follow up but that argument is named wrongly now).

Reply:
hmm, actually `computeEnabledFeatures` is called from a few other places - I think we should actually do a rename for now to reduce the functional changes in the PR

Reply:
I renamed it to `platformSpecificFeatures` for now.